Dockerfile

```diff
@@ -181,6 +194,9 @@ COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/fdtransfer gp
 
 COPY --from=rbspy-builder /tmp/rbspy/rbspy gprofiler/resources/ruby/rbspy
 
+COPY --from=dotnet-builder /usr/share/dotnet /usr/share/dotnet
+COPY --from=dotnet-builder /tmp/.dotnet gprofiler/resources/dotnet
```

Comment:
What's actually required from here? Can you copy only the dotnet-trace binary itself? Does it require any libraries? We want to keep the image size small.